src/libraries/CurrencyDelta.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.20;
+
+import {Currency} from "../types/Currency.sol";
+
+library CurrencyDelta {
+    uint256 constant CURRENCY_DELTA_SLOT = uint256(keccak256("CurrencyDelta")) - 1;
+
+    function _computeSlot(address locker, Currency currency) internal pure returns (bytes32 hashSlot) {
```

Comment:
Do we still want this to be called locker?

Reply:
oh yeah good point